### PR TITLE
P#58329 Bundle settings on one page

### PR DESCRIPTION
**Pull request by hungnc89**
Remove admin page modules

### CODE REVIEW

**Review by jayay — CHANGES_REQUESTED** [review]
Looks good, but could you please also look after the indents? Tabs and spaces might be mixed here.

**Comment by hungnc89** [comment]
> Looks good, but could you please also look after the indents? Tabs and spaces might be mixed here.



> Looks good, but could you please also look after the indents? Tabs and spaces might be mixed here.

I fixed it

**Review by jayay — APPROVED** [review]

**Comment by jayay** [comment]
Looks good, but it'd be nice if the test coverage could go higher

**Review by jayay — APPROVED** [review]

File: readme.txt [hunk 1]
```diff
@@ -73,6 +73,50 @@ Start editing inside the `onoffice-personalized` folder.
 
 == Changelog ==
 
+= 2.20.11 =
```
Comment (jayay) [inline readme.txt:76]: Shouldn't this have been merged during the merge or rebase?

Comment (nglelinh) [reply readme.txt:76]: @jayay sorry, we do not understand your comment, the change is not included in the PR

Comment (jayay) [reply readme.txt:76]: Oh, maybe GitHub included it for some reason then.

**Review by jayay — APPROVED** [review]

**Comment by michaelarnold-dev** [comment]
@nglelinh Could you please increase the code coverage?

**Comment by hungnc89** [comment]
@michaelarnold-dev I wrote more unittest of this PR at #165

**Comment by michaelarnold-dev** [comment]
@hungnc89 It seems like the code coverage could not be increased highly enough. Could you check again? Thank you in advance!

**Review by jayay — APPROVED** [review]

**Pull request merged**